src/common/utxobased/engine/UtxoEngine.ts

```diff
@@ -617,6 +617,13 @@ export async function makeUtxoEngine(
         targets = []
       }
 
+      // Determine whether the transaction should enable RBF given that the
+      // currency supports RBF.
+      const enableRbf =
```

Comment:
`const enableRbf = currencyInfo.canReplaceByFee === true ? edgeSpendInfo.enableRbf ?? spendInfoOtherParams.enableRbf ?? true : false` to preserve legacy behavior when the new flag is missing.